ontology/yaml/resources/HVAC/entity_types/DMP.yaml

```diff
@@ -83,6 +83,15 @@ DMP_EDBPC_ZTM:
   - EDBPC
   - ZTM
 
+DMP_MFSD:
+  description: "Motorized fire smoke damper."
+  is_canonical: true
+  implements:
+  - DMP
+  - MFSD
+  uses:
+  - motorized_damper_position_command
```

Comment:
![MFSD](https://user-images.githubusercontent.com/21256796/224619458-8c625efe-0719-4cbf-bc3f-31d48f24c1ef.png)

sir, can I ask if we can create damper with position (open/closed) instead of percentage? It seems we have a different kind of dampers here that only sends open/close status (also it controls dampers only by open/close commands)